sample_attributes.yml

```diff
@@ -4,9 +4,10 @@ registry_cert_path: /etc/docker/certs.d
 registry_name: /etc/docker/certs.d/registry_hostname:port
 registry_ca_file: /etc/docker/certs.d/registry_hostname:port/ca.crt
 container_user: vagrant
-container_capadd: null
+container_capadd: 'NET_ADMIN,SYS_ADMIN'
```

Comment:
Should the example be different from the default? E.g just use 'NET_ADMIN'?

Reply:
nice idea, done!